web/controllers/notes_controller.ex

```diff
@@ -2,6 +2,13 @@ defmodule Notox.NoteController do
   use Notox.Web, :controller
   alias Notox.Note
 
+  def index(conn, %{"search" => %{"for" => keyword}}) do
```

Comment:
Looks like you can't do that either. You could write the form without a form helper (i.e. just use HTML). Or you could leave it grouped. If so I'd probably make it `%{"search" => %{"query" => query}}`